test_runner/regress/test_metric_collection.py

```diff
@@ -24,13 +24,6 @@
 from werkzeug.wrappers.request import Request
 from werkzeug.wrappers.response import Response
 
-
-@pytest.fixture(scope="session")
-def httpserver_listen_address(port_distributor: PortDistributor):
-    port = port_distributor.get_port()
-    return ("localhost", port)
-
-
```

Comment:
I wonder if this went wrong way in rebase? @LizardWizzard probably changed this.

Reply:
`main` has these lines: https://github.com/neondatabase/neon/blob/b6c7c3290f795d918f726de2df0015bb4a3cd260/test_runner/regress/test_metric_collection.py#L28-L31

Reply:
No, it's not a recent change.

Reply:
No, I moved this out of `test_metric_collection.py` into `neon_fixtures.py`. He didn't change this.

Reply:
Uh ok ... Unsure if this is related? I missed how.

Reply:
Reason was that the added test wants to have an metric endpoint to cause metric task to query synthetic sizes in it's normal fashion.